gruntfile.js

```diff
@@ -55,6 +55,29 @@ module.exports = function (grunt) {
   grunt.registerTask('run-dev', ['build', 'shell:run', 'watch:build']);
   grunt.registerTask('clear', ['clean:app']);
 
+  // Load content of `vendor.yml` to src.jsVendor, src.cssVendor and src.angularVendor
+  grunt.registerTask('vendor', 'vendor:<minified|regular>', function(min) {
+      // Argument `min` defaults to 'minified'
+      var minification = (min === '') ? 'minified' : min;
+      var vendorFile = grunt.file.readYAML('vendor.yml');
+      for (var List in vendorFile) {
+          if (vendorFile.hasOwnProperty(type)) {
+              var thisList = vendorFile[List][minification];
```

Comment:
I don't think that you need to prefix `this` to variable names. Does not really help reading the code.

Use `list` here for example.

Below, you could use `for (var file in list) {` which is a lot more readable and self explanatory.